Yosemite/Yosemite/PointOfSale/POSItemProvider.swift

```diff
@@ -24,5 +24,13 @@ extension POSItem {
 }
 
 public protocol POSItemProvider {
-    func providePointOfSaleItems() async throws -> [POSItem]
+    func providePointOfSaleItems(pageNumber: Int) async throws -> [POSItem]
+}
+
+// Default implementation for convenience, so we do not need to pass the first page explicitely
+// if no pageNumber is given.
+extension POSItemProvider {
+    func providePointOfSaleItems(pageNumber: Int = 1) async throws -> [POSItem] {
+        try await providePointOfSaleItems(pageNumber: 1)
```

Comment:
nit: might we still want to pass the `pageNumber` with a default value 1?

```suggestion
        try await providePointOfSaleItems(pageNumber: pageNumber)
```

Reply:
Agree! I was thinking in the context of having 1 page only so far, but there's no reason to not pass the param down. Updated: https://github.com/woocommerce/woocommerce-ios/pull/14187/commits/f63c91e85036e80d9751875aacf8d614bbd595ac